src/interface.py

```diff
@@ -97,30 +97,40 @@ def get_user_inputs():
         has_question_bank = True
     else:
         has_question_bank = False
+    
+    include_questions_in_pdf = input('Would you like questions to appear in the output PDFs? [y/n]: ')
+
+    if include_questions_in_pdf == 'y' or include_questions_in_pdf == 'Y':
```

Comment:
`if include_questions_in_pdf.upper() == 'Y'`

simplified logic?

Reply:
Nice, I like it!

Reply:
Implemented